messaginginapp/src/test/java/io/customer/messaginginapp/ui/InAppMessagingIntegrationTest.kt

```diff
@@ -0,0 +1,63 @@
+package io.customer.messaginginapp.ui
+
+import io.customer.commontest.extensions.flushCoroutines
+import io.customer.commontest.util.ScopeProviderStub
+import io.customer.messaginginapp.gist.data.model.Message
+import io.customer.messaginginapp.state.InAppMessagingAction
+import io.customer.messaginginapp.state.InAppMessagingManager
+
+object InAppMessagingIntegrationTest {
```

Comment:
Same as above. The `Test` prefix looks like it is a test. Can we please rename it with something like `InAppMessagingIntegrationUtil` etc.?

Reply:
updated